scripts/pkg/scripts/pre_remove.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+# Description:
+# Pre-remove script to stop "<%= product %>".service if it's running.
+
+set -e
+
+echo start pre remove
+
+echo -n "Stopping <%= product %> service..."
```

Comment:
Move inside the `if` or add another `echo` when the service is actually stopping.

Reply:
Moved now.